pkg/utils/common.go

```diff
@@ -173,6 +173,12 @@ func ConvertCommitHashToShort(hash string) string {
 	return hash[:7]
 }
 
+func BuildCommitURL(url string, commitHash string) string {
+	url = strings.Replace(url, "ssh://git@", "https://", 1)
+	url = strings.Replace(url, ".git", "", 1)
+	return fmt.Sprintf("%s/commit/%s", url, commitHash)
```

Comment:
This works for both Github and Gitlab?

Reply:
No it doesnt.  There is work needed to normalize gitlab url when we support it. So that work could effect whats done here

Reply:
worked it out with yiannis and now it supports gitlab